modules/monitoring/application/controllers/TacticalController.php

```diff
@@ -103,4 +114,15 @@ public function indexAction()
             ->render();
         $this->view->statusSummary = $summary;
     }
+
+
+    /**
+     * Return all tabs for this controller
+     *
+     * @return Tabs
+     */
+    private function createTabs()
+    {
+        $this->getTabs()->extend(new OutputFormat(['pdf']));
```

Comment:
Why no PDF export? Isn't that the most desired use here?

Reply:
That's mostly because I just wanted it to be built in quickly, and for some reason the pdf format is horribly broken here...

Reply:
Then please investigate this further. I see absolutely no reason for this change without PDF support. (Whether it produces fancy results or not)

Or simply hide the export functionality in the UI, but only if you need the csv/json functionality for other reasons. ;)